acceptance-tests/lib/testState.ts

```diff
@@ -0,0 +1,65 @@
+import { getTestConfig } from './env';
+import { createCli } from './cmd';
+import { CLI, TestConfig } from './types';
+import { getInitPromptSequence } from './prompt';
+import { existsSync, readFileSync, writeFileSync } from 'fs';
+import yaml from 'js-yaml';
+import rimraf from 'rimraf';
+import { v4 as uuidv4 } from 'uuid';
+
+export class TestState {
+  public config: TestConfig;
+  public cli: CLI;
+  private testConfigFileName: string;
+  private parsedYaml: ReturnType<yaml.load>;
+
+  constructor() {
+    this.config = getTestConfig();
+    this.cli = createCli(this.config);
+    this.testConfigFileName = `hs-acceptance-test.config-${uuidv4()}.yml`;
+  }
+
+  getPAK(): string | undefined {
+    return this.config?.personalAccessKey;
```

Comment:
I'm seeing a few TS errors for invocations of this because some utils require string and this returns string | undefined. One example is [here](https://github.com/HubSpot/hubspot-cli/pull/1181/files#diff-a585f4efe83b1715f0e5085c2e8bec8e25c2fb3a78d31ab21c5ace7cc24b210dR23).

Reply:
https://github.com/HubSpot/hubspot-cli/pull/1181/commits/e0704fd48bdcfcb72c5a1bc0bcfa92dfd98b6c1f